internal/core/src/segcore/SegmentGrowingImpl.cpp

```diff
@@ -698,7 +695,7 @@ SegmentGrowingImpl::bulk_subscript_impl(FieldId field_id,
         // after the above check but before we grabbed the lock, we should grab
         // from index as the data in chunk may have been removed in
         // try_remove_chunks.
-        if (!indexing_record_.SyncDataWithIndex(field_id)) {
```

Comment:
If the index data does not catch up with the insert data (the first build is completed, but the add is not completed), will there be a problem here?

Reply:
indexing_record_.HasRawData(field_id) include indexing_record_.SyncDataWithIndex(field_id).